src/caffe/util/im2col.cpp

```diff
@@ -201,5 +206,4 @@ template void col2im_nd_cpu<double>(const double* data_col,
     const int* kernel_shape, const int* pad, const int* stride,
     const int* dilation, double* data_im);
 
-
```

Comment:
Please don't include whitespace changes in a commit that changes code somewhere else. (It's okay to make whitespace fixes, but those should go in their own commits unless they're adjacent to modified code.)